src/components/Menu/keys.js

```diff
@@ -0,0 +1,24 @@
+const isMac = (navigator.userAgent.includes('Mac'))
+
+const MODIFIERS = {
+	Mod: isMac ? 'Meta' : 'Control',
+	Alt: 'Alt', // Option key, on Apple computers.
+	Control: 'Control',
+	Shift: 'Shift',
+
+	// unused
+	// AltGraph: 'AltGraph',
+	// Meta: 'Meta', // Command key on Apple computers
+}
+
+const TRANSLATIONS = {
+	[MODIFIERS.Mod]: t('text', isMac ? 'Command' : 'Control'),
+	[MODIFIERS.Control]: t('text', 'Ctrl'),
```

Comment:
Is it on purpose that `MODIFIERS.Control` is translated to 'Ctrl' while `MODIFIERS.Mod` is translated to `Control`? Also, if I'm not mistaken, then `MODIFIERS.Control` is not used at all in the code, only `MODIFIERS.Mod`, right?

Reply:
Additionally when using `ctrl` for the key, which is fine to make it clear the key is meant, why not also using the abbrev for the command key too? (`cmd`)

Reply:
`Control` key exist in Mac's keyboard.
So we can directly address it.

But, tiptap change `Ctrl` to `Cmd` natively.

https://tiptap.dev/api/keyboard-shortcuts

> You can use Mod as a shorthand for Cmd on Mac and Control on other platforms.